lib/miq_automation_engine/service_models/miq_ae_service_generic_object.rb

```diff
@@ -14,7 +16,15 @@ def remove_from_service(service)
 
     private
 
+    def ae_user_identity
+      ae_user = DRb.front.workspace.ae_user
+      group   = ae_user ? ae_user.current_group : nil
+      tenant  = ae_user ? ae_user.current_tenant : nil
+      ar_method { @object.ae_user_identity(ae_user, group, tenant) }
+    end
+
     def method_missing(method_name, *args)
+      ae_user_identity
```

Comment:
do we need to setup identity every time? `method_missing` might be called frequently.